h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -492,11 +519,33 @@ def standalone_test(test):
 
     h2o.log_and_echo("------------------------------------------------------------")
     h2o.log_and_echo("")
-    h2o.log_and_echo("STARTING TEST")
+    h2o.log_and_echo("STARTING TEST "+test.__name__)
     h2o.log_and_echo("")
     h2o.log_and_echo("------------------------------------------------------------")
     test()
 
+def run_tests(tests, run_in_isolation=True):
```

Comment:
simplistic "test suite" runner: a bit frustrated that when creating Py test files with multiple tests, those don't run is isolation by default.